twisted/web/_http2.py

```diff
@@ -577,8 +577,13 @@ def _handleWindowUpdate(self, event):
         streamID = event.stream_id
 
         if streamID:
-            # Update applies only to a specific stream. If we don't have the
-            # stream, that's ok: just ignore it.
+            # Update applies only to a specific stream. We may have already
+            # cleaned up our stream state, making this a late WINDOW_UPDATE
+            # frame. That's fine, it's not a problem: the update is unnecessary
+            # but benign. We'll just ignore it.
+            if not self._streamIsActive(streamID):
+                return
```

Comment:
I think that the 'We might have a late WINDOWS_UPDATE` comment should be in this block.

For me `if not self._streamIsActive(streamID):` speak for itself and we no longer need the verbose comment above

`Update applies only to a specific stream. We may have already cleaned up our stream state`